teps/0084-endtoend-provenance-collection.md

```diff
@@ -153,14 +163,38 @@ In this proposal, we are suggesting following extensions for an end-to-end and c
 As the event-payload is received by the `EventListener`, we need an interface to introspect/query payload so we can sign it from the controller.
 
 ##### 2. Provenance for pipelinerun
-In addition to individual `taskruns`, we collect the attestation for `pipelineruns`. This would allow us to attest: (a) list of all tasks executed in the pipelines (b) order in which these tasks were executed ( c) list of shared resources created/shared across tasks
+In addition to individual `taskruns`, we collect the attestation for `pipelineruns`. This would allow us to attest: (a) list of all tasks executed in the pipelines (b) order in which these tasks were executed (c) list of parameters shared across tasks.
+
+Chains supports two attestation formats for `taskruns`: `tekton` and `in-toto`. Both must be supported for `pipelineruns` attestations.
+
+The `tekton` format is simply the `Status` of a `taskrun`. Similarly, for `pipelineruns`, this should also be the `Status` of the `pipelinerun`.
+
+The `in-toto` format created by chains wraps the [SLSA Provenance v0.2](https://slsa.dev/provenance/v0.2) attestation format. The `in-toto` attestation itself should be the same for `taskruns` and `pipelineruns`. The `SLSA Provenance` attestation requires special attention:
+
+* **subject**: For `taskruns`, Chains processes the `taskrun` results and resources to determine which images were built then use this information to populate this field. For `pipelineruns`, Chains must instead process the `pipelinerun` results. Pipeline authors can easily set `pipelinerun` results based on `taskrun` results, allowing them to more easily use existing Tasks that do not conform to Chains type hinting while keeping the Chains implementation simpler (no deep inspection). Given that resources are deprecated, further support should not be added.
+* **predicate.buildType**: Chains sets this value to `https://tekton.dev/attestations/chains@v2` for `taskruns` attestations. In order to facilitate consumption, a different value should be used: `https://tekton.dev/attestations/chains/pipelinerun@v2`.
+* **predicate.invocation**: For `taskruns`, Chains populates this object with the task parameters. Similarly, Chains must populate this object with `pipelinerun` parameters. Default parameters must also be added.
```

Comment:
I was assuming these are all resolved parameters but the example attestation includes mapping 🤔 :

https://github.com/lcarva/minimal-container/blob/178df1bfeb1b089bd22695f75aa99fd2d4fa679e/tekton/examples/pipelinerun-attestation-tep-84.json#L62